### DISCUSSION TITLE
How to switch space and tab in zmk?

### DISCUSSION THREAD

**Discussion by LeducH** [manna-harbour/miryoku#269] [Q&A] [answered]
I like to switch the keys `tabs` and `space`. 

I love the layout and I just like change  `tabs` and `space`  I am using a corne with a nice nano and nice view. 

how can it be changed?

**Comment by manna-harbour** [comment, marked as answer]
To customise, see https://github.com/manna-harbour/miryoku_zmk#customisation. You can substitute a custom Base layer. Also see https://github.com/manna-harbour/miryoku/discussions/224#discussioncomment-6323631.

However, the layout is designed with the middle thumb key as the primary thumb key. If that's not comfortable you might need to use a different keyboard.

**Reply by LeducH** [reply]
Uff shoots fired. Yeah I am new to split and now got used to your layout. When I started the space key looked like it should be switched but after getting used to it I would say your did a amazing job.  

I see you in a while.

**Reply by manna-harbour** [reply]
Ha, to be fair, the Corne thumb keys are a little cramped! Also, it might look like the 1.5u thumb keys are supposed to be home, but that's really just a historical accident.

But the resting thumb position does vary person to person, and keyboard thumb positions can also vary a lot. With some practice you can generally adapt to a pretty wide range of positions, but it's worth trying to find a board that is a good match for your hands.
